dev/integration_tests/ios_host_app/Host.xcodeproj/project.pbxproj

```diff
@@ -329,14 +340,15 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 11.4;
-				MTL_ENABLE_DEBUG_INFO = YES;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```

Comment:
Right. The host app is not currently used by any tests, so that is not an immediate problem. In the past I think the bots have been upgraded to match the latest Xcode out there - ensuring test coverage of the tooling developers are transitioning to.

Reply:
Xcode 10 has a couple issues that are being fixed, although I think the bigger ones are on the engine side rather than the Flutter side.  This looks good to me, it should be easy to fix if we do end up wanting to use it on a bot before it's upgraded.  Thanks!